metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/5.6.2/package/scripts/slave.py

```diff
@@ -48,19 +51,30 @@ def slave():
          content=InlineTemplate(params.elastic_env_sh_template)
          )
 
-    configurations = params.config['configurations']['elastic-site']
-
-    File("{0}/elasticsearch.yml".format(params.conf_dir),
+    elastic_site = params.config['configurations']['elastic-site']
+    path = "{0}/elasticsearch.yml".format(params.conf_dir)
+    Logger.info("Cre")
```

Comment:
I don't think "Cre" is what you wanted here, typo?

Reply:
Hm, adding to my list of cleanup tasks.